src/BigInteger/Adapter/Bcmath.php

```diff
@@ -313,7 +313,14 @@ public function baseConvert($operand, $fromBase, $toBase = 10)
             $decimal = '0';
             for ($i = 0, $len  = strlen($operand); $i < $len; $i++) {
                 $decimal = bcmul($decimal, $fromBase);
-                $decimal = bcadd($decimal, strpos($chars, $operand[$i]));
+
+                if ($fromBase <= 36) {
+                    $remainder = base_convert($operand[$i], $fromBase, 10);
+                } else {
+                    $remainder = strpos($chars, $operand[$i]);
```

Comment:
Could we do these as a ternary instead?

```php
$remainder = ($fromBase <= 36)
    ? base_convert($operand[$i], $fromBase, 10)
    : strpos($chars, $operand[$i]);
```

I feel the ternary makes it more clear that we're creating the `$remainder` variable, from one of two alternatives.